src/content/admin/okteto-insights.mdx

```diff
@@ -80,7 +80,7 @@ Okteto will progressively make more data available through this endpoint. The fo
     - `pod_name`: It contains the name of the pod being represented.
     - `namespace_name`: It contains the namespace name.
     - `node_name`: It contains the node name.
-- `okteto_system_pod_container_memory_requests_bytes`: This metric is a gauge which represents the memory requested for a specific pod and standard container at that point in time. It has the following labels:
+- `okteto_system_pod_container_memory_requests_bytes`: This metric is a gauge which represents the memory [requested](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) for a specific pod and standard container at that point in time. It has the following labels:
```

Comment:
@codyjlandstrom should we provide the same link to `gauge` like we do in the first metric for consistency? Since we link out on `requested` for each entry it might be good to do the same for `gauge` for consistency and overcommunication.

Non-blocking suggestion

Reply:
Yes, good call. Agree it would make it more consistent and remove any potential confusion.